### PR TITLE
[NeuralChat] Retrieval pdf figure to text

### PR DESCRIPTION
**Pull request by Liangyx2**
Enable a more efficient and accurate way to parse text and image.

### CODE REVIEW

**Comment by XuhuiRen** [comment]
recommend adding a label to distinguish that using the naive PDF to text and OCR pdf to text

**Comment by XuhuiRen** [comment]
add ut

**Review by XuhuiRen — APPROVED** [review]

File: intel_extension_for_transformers/neural_chat/pipeline/plugins/retrieval/requirements.txt [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 chromadb==0.4.15
+easyocr
 InstructorEmbedding
 langchain==0.0.354
 langchain_core==0.1.18
 langid
 markdown
 openpyxl
+PyMuPDF
 PyPDF2
```
Comment (XuhuiRen) [inline intel_extension_for_transformers/neural_chat/pipeline/plugins/retrieval/requirements.txt:11]: delete Pypdf2

**Review by hshen14 — APPROVED** [review]

**Comment by hshen14** [comment]
LGTM

**Pull request merged**